src/test/regress/sql/mx_application_name.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SCHEMA mx_app_name;
+
+CREATE TABLE output (line text);
+
+-- a hack to run command with another user
```

Comment:
Isn't this the same user but another application name?

Reply:
yes, copy & paste error